### PR TITLE
Issue/2856 remove public private

### PR DESCRIPTION
**Pull request by stellanl**
#### Before the PR (what is the issue or what needed to be done)
Public/private not used by partners.
#### The solution
Remove functionality.

#### Reviewer Checklist
* [ ] Added an explanation about the work done
* [ ] Connected the PR and the issue on Zenhub
* [ ] Added a test plan to the issue
* [ ] Updated the copyright header (when relevant)
* [ ] Formatted the code
* [ ] Added a documentation (if relevant)
* [ ] Added some unit tests (if relevant)

### CODE REVIEW

File: GAE/src/com/gallatinsystems/survey/domain/SurveyGroup.java [hunk 1]
```diff
@@ -166,15 +166,15 @@ public void setDefaultLanguageCode(String defaultLanuageCode) {
         this.defaultLanguageCode = defaultLanuageCode;
     }
```
Comment (valllllll2000) [inline GAE/src/com/gallatinsystems/survey/domain/SurveyGroup.java]: Shall we maybe remove commented out code?

File: GAE/src/org/waterforpeople/mapping/app/gwt/client/survey/SurveyDto.java [hunk 2]
```diff
@@ -33,7 +33,7 @@
     private List<QuestionGroupDto> questionGroupList;
     private String path;
     private Long surveyGroupId = null;
-    private String pointType;
+//    private String pointType;
```
Comment (valllllll2000) [inline GAE/src/org/waterforpeople/mapping/app/gwt/client/survey/SurveyDto.java]: Same here, maybe remove commented out code on this file

File: GAE/src/com/gallatinsystems/survey/domain/SurveyGroup.java (hunk 1 above)
Comment (stellanl) [reply GAE/src/com/gallatinsystems/survey/domain/SurveyGroup.java]: Certainly.

File: GAE/src/org/waterforpeople/mapping/app/gwt/client/survey/SurveyDto.java (hunk 2 above)
Comment (stellanl) [reply GAE/src/org/waterforpeople/mapping/app/gwt/client/survey/SurveyDto.java]: Definitely

**Pull request merged**